### ISSUE TITLE
Memory Connector: inq filter should respect Arrays

### ISSUE THREAD

**Issue by fabien** [strongloop/loopback-datasource-juggler#147]
See: https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/connectors/memory.js#L379

To make the comparison follow MongoDB for example, the value should intersect at least once (by casting any value to an array). 

Something like (needs a more fitting and well-tested solution, without dependencies):

``` javascript
if (example.inq) {
  if (!value) return false;
  var sample = [].concat(value || []);
  return _.intersect(example.inq, sample).length > 0;
}
```

**Comment by loay** [comment]
Hi @fabien 
Did the merged pull request #710 Fix the issue?
Thanks.

**Comment by rmmlxr** [comment]
Does not work for me; using postgres connector.
